users/models.py

```diff
@@ -1,6 +1,6 @@
 # from django.db import models
-from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.models import AbstractUser, PermissionsMixin
 
 
-class User(AbstractUser):
-    pass
+class User(AbstractUser, PermissionsMixin):
+    ...
```

Comment:
I go back and forth on this one (`...` vs `pass`). We should probably make a distinction for the project about when to use one vs the other. I think I'll put `pass` back for now.

Reply:
This could be a good dev brainstorming discussion to understand the differences :)